.docker-config/traefik/traefik-provider.toml

```diff
@@ -37,7 +37,7 @@
     [[http.services.api.loadBalancer.servers]]
       url = "http://localhost:5000"
     [[http.services.publikator-api.loadBalancer.servers]]
-      url = "http://localhost:5010"
```

Comment:
Wouldn't it make sense to just merge the following services: `api` and `publikator-api`

Reply:
Traefik is only used for development and also there only by me atm. As it does no harm exposing the backend on two ports I'm going to leave it for now.